src/main/java/org/folio/service/FileServiceImpl.java

```diff
@@ -16,45 +16,56 @@
 public class FileServiceImpl implements FileService {
 
   private Vertx vertx;
-  private FileDao FileDao;
+  private FileDao fileDao;
 
-  public FileServiceImpl(FileDao FileDao) {
-    this.FileDao = FileDao;
+  public FileServiceImpl(FileDao fileDao) {
+    this.fileDao = fileDao;
   }
 
   public FileServiceImpl(Vertx vertx, String tenantId) {
     this.vertx = vertx;
-    FileDao = new FileDaoImpl(vertx, tenantId);
+    fileDao = new FileDaoImpl(vertx, tenantId);
   }
 
   public Future<List<File>> getFiles(String query, int offset, int limit) {
-    return FileDao.getFiles(query, offset, limit);
+    return fileDao.getFiles(query, offset, limit);
   }
 
   @Override
   public Future<Optional<File>> getFileById(String id) {
-    return FileDao.getFileById(id);
+    return fileDao.getFileById(id);
   }
 
   @Override
-  public Future<String> addFile(File file, DataImportUploadFilePostMultipartFormData data) {
+  public Future<List<File>> getFileByUploadDefinitionId(String id) {
+    return fileDao.getFileByUploadDefinitionId(id);
+  }
+
+  @Override
+  public Future<String> addFile(File file) {
```

Comment:
Will this service be able to add multiple list of files at one call ?
Or the user of this interface should call addFile method per each batch upload ?